cosi.go

```diff
@@ -11,14 +11,14 @@ import (
 
 	"os"
 
-	"gopkg.in/dedis/cothority.v0/lib/dbg"
+	"github.com/dedis/cothority/lib/dbg"
 )
 
-// Name of the binary
+// BinaryName represents the Name of the binary
 const BinaryName = "cosi"
 
 // Version of the binary
-const Version = "0.8.4"
+const Version = "0.8.6"
```

Comment:
Do we even care ? Your bash script `cross_compile.sh` does it for us no ? We should not *set* that variable and either use your script to set it to the right value, or either use the built-in compilation flags. `-ldflags ....`